doc/advanced_configuration.rst

```diff
@@ -77,6 +77,39 @@ starting with ``plot_long_examples_``, you would do:
 As the patterns are parsed as `regular expressions`_, users are advised to consult the
 `regular expressions`_ module for more details.
 
+.. _sub_gallery_order:
+
+Sorting the subsections inside a gallery
+========================================
+
+Gallery subsections are sorted by default alphabetically by their folder
+name, and as such you can always organize them by changing your folder
+names. An alternative option is to use a sortkey to organize those
+subsections. We provide an explicit order sortkey where you have to define
+the order of all subfolders in your galleries.
+
+.. code-block:: python
+
+    from sphinx_gallery.sorting import ExplicitOrder
+    sphinx_gallery_conf = {
+        'examples_dirs': ['../examples','../tutorials'],
+        'subsection_order': ExplicitOrder(['../examples/sin_func',
```

Comment:
Could we add one sentence just pointing out the fact that even though there are multiple examples_dirs, there is only one `ExplicitOrder` key given to subsection_order? I know that people could infer this from looking at the folder names in the sorting key, but I think it might confuse some folks who assume you'd need to give two `ExplicitOrder` keys, one for each gallery folder.